using-timescaledb/backup.md

```diff
@@ -52,6 +52,19 @@ SELECT timescaledb_pre_restore();
 SELECT timescaledb_post_restore();
 ```
 
+>:WARNING: Using the flag `-j` with `pg_restore` to restore using
+>multiple workers is not supported and might generate errors. You can
+>use it if you first restore the `_timescaledb_catalog` schema without
+>the `-j` option, for example:
+>```
+>pg_restore -Fc -n _timescaledb_catalog -d tutorial tutorial.bak
```

Comment:
It's unclear whether you can do this, or if you need to first do a pre-data restore of everything, then restore the data to the `_timescaledb_catalog`, then data for everything else in parallel, then you can do a parallel post-data restore. That's what I do in the backup restore tool. I think the reason I did that is that I think there can be issues restoring things to the catalog when the chunks we're referring to don't exist, and if we ever decide to do OIDs in the catalog, things will definitely get weird...but if you tested this and it doesn't throw weird errors, this should be fine, it's definitely simpler to explain.

Reply:
No, you were right, this did not work. I updated the instructions and have tested this sequence, which works.